packages/cli/create-strapi-app/src/cloud.ts

```diff
@@ -16,13 +15,13 @@ function assertCloudError(e: unknown): asserts e is CloudError {
   }
 }
 
-export async function handleCloudProject(projectName: string): Promise<void> {
+export async function handleCloudProject(): Promise<void> {
```

Comment:
```suggestion
export async function handleCloudLogin(): Promise<void> {
```
Probably better to rename the function to something more representative